client/cmd/mmbot/main.go

```diff
@@ -0,0 +1,215 @@
+package main
+
+/*
+Create core, wallets and register for server using dexc before starting the bot.
+*/
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	"decred.org/dcrdex/client/core"
+	"decred.org/dcrdex/dex"
+	"decred.org/dcrdex/server/admin"
+
+	_ "decred.org/dcrdex/client/asset/btc"
+	_ "decred.org/dcrdex/client/asset/dcr"
+)
+
+var (
+	userDir, _            = os.UserHomeDir()
+	defaultConfigFileName = "mm.json"
+	defaultCoreDir        = filepath.Join(userDir, ".dexc")
```

Comment:
Why not dcrutil.AppData?  Using home/.dexc isn't going to jive with Windows and probably macos